src/types/commons.ts

```diff
@@ -41,3 +44,37 @@ export const CommaSeparatedListOf = (decoder: t.Mixed) =>
       ),
     String
   );
+
+/**
+ * Parses a string into a deserialized json
+ */
+export type JSONFromString = t.TypeOf<typeof jsonFromString>;
+// eslint-disable-next-line @typescript-eslint/ban-types
+export const jsonFromString = new t.Type<object, string>(
+  "JSONFromString",
+  t.UnknownRecord.is,
+  (m, c) =>
+    t.string.validate(m, c).chain(s =>
+      E.tryCatch2v(
+        () => t.success(JSON.parse(s)),
```

Comment:
what about use parse from `fp-ts/lib/Json` ?

Reply:
Json module has been added starting from v2.10.0, we are using v1.17.x here :(